db.sql.editor/test/unit/src/org/netbeans/modules/db/sql/editor/completion/SelectCompletionQueryTest.java

```diff
@@ -143,27 +143,19 @@ public void testCompletion() throws Exception {
                     }
                 }
             }
-        } finally {
-            reader.close();
         }
         String sql = sqlData.toString();
         Metadata metadata = TestMetadata.create(modelData);
         if (stdout) {
             performTest(sql, metadata, System.out);
         } else {
             File result = new File(getWorkDir(), getName() + ".result");
-            Writer writer = new BufferedWriter(new OutputStreamWriter(new FileOutputStream(result), "utf-8"));
-            try {
+            try (Writer writer = new BufferedWriter(new OutputStreamWriter(new FileOutputStream(result), "utf-8"))) {
                 performTest(sql, metadata, writer);
-            } finally {
-                writer.close();
             }
             File pass = new File(getWorkDir(), getName() + ".pass");
-            InputStream input = SelectCompletionQueryTest.class.getResource(getName() + ".pass").openStream();
-            try {
-                copyStream(input, pass);
-            } finally {
-                input.close();
+            try (InputStream input = SelectCompletionQueryTest.class.getResource(getName() + ".pass").openStream()) {
```

Comment:
Changing this was not necessary. But if we do it, why not `getResourceAsStream`?